requirements.txt

```diff
@@ -7,7 +7,7 @@ Flask-DebugToolbar==0.10.1
 Flask-Login==0.4.1
 Flask-SQLAlchemy==2.3.2
 itsdangerous==0.24
-Jinja2==2.10
```

Comment:
Is this a patch update? What is a dependency? Does this mean I can never go back to hosting my own project? I have to put the audio over this project. Oh gosh, I just liked the image in the background and the neon colors made me think of an arcade.